.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Related Issues
+<!-- link all issues that this PR might resolve/fix. If an issue doesn't exist, include a brief motivation for the change being made.-->
+
+## Proposed Changes
+<!-- provide a clear list of the changes being made-->
+
+
+## Additional Info
+<!-- callouts, links to documentation, and etc-->
+
+## Checklist
+
+Before you mark the PR ready for review, please make sure that:
+- [ ] The PR title is in the form of of `<PR type>: <#issue number> <area>: <change being made>`
+    - example: ` fix: #1234 mempool: Introduce a cache for valid signatures`
+    - `PR type`: _fix_, _feat_, _BREAKING CHANGE_, _build_, _chore_, _ci_, _docs_, _perf_, _refactor_, _revert_, _style_, _test_
```

Comment:
misc?

Reply:
whats the difference between chore and misc?

Reply:
i would think of something like "fix lint" as a chore.

Reply:
Not sure what a _BREAKING CHANGE_ would count as

Reply:
> i would think of something like "fix lint" as a chore.

and misc?